BP/common/culture/cultures/00_central_semitic.txt

```diff
@@ -0,0 +1,1140 @@
+		}
+
+		dynasty_names = {
+			"dynn_Rasulid"
+			"dynn_Umarid"
+			"dynn_Najadid"
+			"dynn_Dinnid"
+			"dynn_Qutalmish"
+			"dynn_Qadir"
+			"dynn_Toghan"
+			"dynn_Walashma"
+			"dynn_Usfurid"
+			"dynn_Jarwanid"
+			"dynn_Zurayid"
+			"dynn_Muazzamid"
+			"dynn_Banu_Hatim"
+			"dynn_Qubaybid"
+			"dynn_Halabanid"
+			"dynn_Basri"
+			"dynn_Hallaj"
+			"dynn_Baghdadi"
+			"dynn_Bistami"
+			"dynn_Karkhi"
+			"dynn_Qadir"
+			"dynn_Jilani"
+			"dynn_Chishti"
+			"dynn_Badr"
+			"dynn_Miqadiyid"
+			"dynn_Munirid"
+			"dynn_Nabhanid"
+			"dynn_Nadjafid"
+			"dynn_Hamdan"
+			"dynn_Ashhadid"
+			"dynn_Quraishid"
+			"dynn_Kamalid"
+			"dynn_Mosulid"
+			"dynn_Jazrid"
+			"dynn_Husayn"
+			"dynn_Nazeerid"
+			"dynn_Nasrid"
+			"dynn_Bichrid"
+			"dynn_Hafizid"
+			"dynn_Fakhourid"
+			"dynn_Nawawid"
+			"dynn_Naderid"
+			"dynn_Chakroun"
+			"dynn_Atrashid"
+			"dynn_Shehhid"
+			"dynn_Hasanid"
+			"dynn_Qasimid"
+			"dynn_Sulaymanid"
+			"dynn_Halwasid"
+			"dynn_Jamalid"
+			"dynn_Abbasid"
+			"dynn_Razinid"
+			"dynn_Numayrid"
+			"dynn_Kaabid"
+			"dynn_Jabrid"
+			"dynn_Chrusarkha"
+			"dynn_Hakamid"
+			"dynn_Hanaya"
+			"dynn_Ibrahimid"
+			"dynn_Al-Duhli"
+			"dynn_Kinda"
+			"dynn_Madhhij"
+			{ "dynnp_al-" "dynn_Zawayhi" }
+			"dynn_Mufaddalid"
+			"dynn_Zurayid"
+			"dynn_Bannu_Man"
+			"dynn_Mahdid"
+			"dynn_Hatimid"
+			"dynn_Sulaymanid"
+			"dynn_Tanukh"
+			"dynn_Hilalid"
+			{ "dynnp_al-" "dynn_Riyahdid" }
+			{ "dynnp_al-" "dynn_Athbejid" }
+			{ "dynnp_al-" "dynn_Zoghbadid" }
+			{ "dynnp_al-" "dynn_Adid" }
+			"dynn_Suleimid"
+			{ "dynnp_al-" "dynn_Zegbid" }
+			{ "dynnp_al-" "dynn_Debbabid" }
+			{ "dynnp_al-" "dynn_Awfid" }
+		}
+
+		male_names = {
+			Abu-Bakr Aarif Abdul-Gafur Abdul-Jaleel Abdul-Qadir Abdul-Wahab Abdul Abdullah Aghlab Akin Ali Alim Aram Azam Bahir Burhanaddin
+			Fadil Fadl Faruk Ghalib Hafiz Halil Hasan Husam Hussayn Ibrahim Idris Is_mail Isa Jabir Jalil Jibril Khaireddin Khalil Mahdi Mahmud
+			Mansur Mirza Mubarak Muhammad Mukhtar Murad Musa Muslihiddin Muzaffaraddin Najib Nasr Nizam Qawurd Ramadan Sa_daddin Sadiq Sami Samir Shaiban
+			Shamir Shujah Sulayman Talib Ubayd Uways Yahya Ya_qub Youkhanna Yusuf Zeyd
+		}
+		female_names = {
+			Adila Amsha Asiya Faghira Habiba Hanifa Jahaira Kamala Layla Maryam Nafisa Nyawela Parand Parween Paymaneh Paywand Qamara Rafiqa Rasa Rashida Reshawna
+			Saaman Sabba Saghar Sahba Sajida Samira Semeah Setara Shahrbano Shahzadah Shameem Shararah Sheeftah Sheeva Shogofa Shokouh Shola Sholah Simin Souzan
+			Taliba Tanaz Taneen Yagana Yakta Yasmin Zaynab
+		}
+		dynasty_of_location_prefix = "dynnp_mn"
+		
+		patronym_prefix_male = "dynnpat_pre_ibn"
+		patronym_prefix_female = "dynnpat_pre_bint"
+		
+		dynasty_title_names = yes
+		founder_named_dynasties = yes
+
+		ethnicities = {
+			10 = arab
+		}
+
+		mercenary_names = {
+			{ name = "mercenary_company_the_wanderers" }
+			{ name = "mercenary_company_bedouin_company_of_place" }
+		}
+	}
+	levantine = {
```

Comment:
This name is misleading as well, same as for Egyptian, Levantine isn't necessarily associated with Arabic.